README.md

```diff
@@ -1,4 +1,87 @@
-demo: https://greedychicken.github.io/movies-browser/
+# Movies Browser 🎥
+
+![headline.png](/src/images/headline.png)
+
+**Movies Browser** is an application that allows users to browse movies and people from film industry. It was created as the final project of the Frontend Developer course at YouCode and was based on the graphic design in Figma. Data about movies and people is fetched from an [external API](https://www.themoviedb.org/).
+
+## Table of content
+
+- [Demo](#demo)
+- [Preview](#preview)
+- [General Information](#general-information)
+- [How to use](#how-to-use)
+- [Tech Stack](#tech-stack)
+- [Authors](#authors)
+- [External Sources](#external-sources)
+- [Available Scripts](#available-scripts)
+
+## Demo
+
+You can find a demo version [here](https://greedychicken.github.io/movies-browser/index#/movies/1).
+
+## Preview
+
+![Preview](...)
+
+## General Information
+
+Application allows to easily browse movies and actors based on **external API**. Entire project is styled with use of **Styled Components**, and it is also adapted to follow **Responsive Web Design Principle**, which makes it fully responsive for all popular devices. In addition, each movie and person has a separate page with information about the crew, cast, and many more.
+
+## How to use
+
+Movies Browser is composed of four main views:
+
+**`Movies page`**
+
+On this page you can browse recently popular movies. Each movie tile includes information such as title, release year, film genres and viewers' rate. To find out more about the specific movie, just click on its tile, and then you will be redirected to a movie subpage.
```

Comment:
", and then you will be redirected to a movie subpage."
Maybe: "to redirect you to a movie subpage"